pkg/state/helmx.go

```diff
@@ -119,6 +119,20 @@ func (st *HelmState) appendCascadeFlags(flags []string, helm helmexec.Interface,
 	return flags
 }
 
+// append show-only flags to helm flags
+func (st *HelmState) appendShowOnlyFlags(flags []string, showOnly []string) []string {
```

Comment:
add tests for this func.

Reply:
The test for this will be in `pkg/state/helmx_test.go` in my next push.